stdlib/runtime/exception.gr

```diff
@@ -1,5 +1,5 @@
 @noPervasives
-@exceptionMod
+@noExceptions
```

Comment:
One tiny NIT, do we just want to add a comment here something like, `// Exceptions cannot be thrown from the exception module or we would have a circular dependency.`?